keylime/ima.py

```diff
@@ -250,91 +255,92 @@ def process_whitelists(wl_data, excl_data):
         tmp = whitelist.get(path,[])
         tmp.append(fhash)
         whitelist[path]=tmp
-    
+
     if whitelist.get('boot_aggregate',None) is None:
         logger.warning("No boot_aggregate value found in whitelist, adding an empty one")
-        whitelist['boot_aggregate'] = [START_HASH.encode('hex')]
-    
+        whitelist['boot_aggregate'] = ['0000000000000000000000000000000000000000']
```

Comment:
Should we switch this back to being a variable, instead of hard-coded?

Reply:
I will need to test this first, I was running into some issues before. If it works ok for you and want to push a patch, go for it. I am just fixing up CI on another branch.